Books/client/src/hooks/useWindowSize.tsx

```diff
@@ -0,0 +1,32 @@
+// source: https://usehooks.com/useWindowSize/
+
+import { useState, useEffect } from 'react';
+// Hook
+export default function useWindowSize() {
```

Comment:
What's the use of this hook?

Reply:
This hook is being used to get window width and height, and it will be also changed if user change his window size. I'm using width from that hook to show different amount of items in slider, depending on screen width.

Reply:
Okay, that makes sense. I think I'll use this hook to for the landing page